Fixes and Clean up

shamalisulakhe opened the pull request:
Changes to-do
- [x] Move the class written in the main file
- [x] Add dependency check
- [x] Folder structure - includes, admin folder
- [x] defined function in a function - for eg `kws_gf_load_functions()`
- [x] Coding standards/formatting
- [x] Use of static/public 
- [x] Update languages
- [x] Putting style CSS
- [x] PHP notices and warnings
- [x] Update readme
- [x] Update version

Commit:
Version 4.1.3

Commit:
Restructure plugin folders and files

Commit:
Moved 1 remaining file to includes

shamalisulakhe commented:
Commit [1137f12](https://github.com/katzwebservices/Gravity-Forms-Directory/commit/1137f12d151d72ad5628f4861f4796fc30dac565)

Changes done - 
1. Moved files to includes folder (need to think of admin folder yet, for admin files)
2. Separated activation and deactivation functions in separate classes
3. Separated shortcode functions in a separate class to reduce lengthy classes
4. Introduced constants for version, plugin path, url, minimum GF version, etc.
5. Added dependancy check (Check if GF is active and if not, deactivate the Directory plugin. This will reduce the need to check for GF at multiple instances.)

Commit:
Move activation action in admin.php to activator class

Commit:
Added deprecated functions to admin.php

Commit:
Reverted deprecation for activation functions

Commit:
Added deprecated functions in GFDirectory

Commit:
remove function kws_gf_load_functions (testing)

Commit:
Renamed admin.php file

shamalisulakhe commented:
[Commit](https://github.com/katzwebservices/Gravity-Forms-Directory/commit/51d259bf5fcbd1d91ede807290b8417c42b50291) 
* Added deprecated functions
* Removed and tried @kws_gf_load_functions@ function
* Renamed admin.php
* Move activation action in admin.php to activator class

Commit:
correct class name

Commit:
Fix activation notice

shamalisulakhe commented:
[Commit](https://github.com/katzwebservices/Gravity-Forms-Directory/pull/36/commits/af9799aaa510e46722ae6ac222d826f8f26fc367) 

Fixed activation function and notice errors
Tested address field
Testing partial search issue #34

Commit:
Issue #34 solution

Commit:
Add template for directory shortcode

Commit:
Move field-ids.php to views

Commit:
Add template for admin settings page

Commit:
Add template for admin mce popup

Commit:
Move template-rows.php to views

Commit:
Move gravityview-info.php to views

Commit:
rename edit-form.php

Commit:
Move Directory colmumn tab HTML to view file

Commit:
Move Directory admin head to view file

Commit:
editor script content to view file

Commit:
Formatting

Commit:
Renamed file names containing class

Commit:
Move select column page content to view file

Commit:
Formatting class-gf-directory-shortcode.php

Commit:
Move entry detail content to view file

Commit:
Formatting class-gf-directory.php

shamalisulakhe commented:
[Commit](https://github.com/katzwebservices/Gravity-Forms-Directory/commit/710b2a2ead04b88b9111e7f3658f22d27fdd9628) 

    Adding views folder and separated HTML content in view files
    Formatting and doc blocks to class-gf-directory.php and class-gf-directory-shortcode.php
    Some minor changes like changing file names and others as observed.

Commit:
Issue #38 solution

Commit:
Fix PHP warning on undefined variable

Commit:
Fix PHP warning on switch case continue

Commit:
Issue #39 solution

Commit:
Formatting

Commit:
Formatting

Commit:
Formatting

Commit:
Create POT file

Commit:
Update README

Commit:
Update version

Commit:
Add docblock for depreacated methods

Commit:
Update docblock version

Commit:
resolves #41 - Change title 'Directory & addons' to 'Directory'

Commit:
resolves #52 correct text [ci skip]

Commit:
resolves #49

Commit:
fixes #47

Commit:
fixes JS var page formatting

Commit:
correct get_base_url

Commit:
Replaced plugins_url() with constant GF_DIRECTORY_URL

Commit:
Update languages

Commit:
fixes #43

Commit:
fixes #48

Commit:
fixes #50

Commit:
fixes #53

andjules commented:
THANK YOU SOOOOO MUCH for your hard and skilled work.
I just installed this branch and it is working on a website I built for a church in Toronto around 3 years ago.
I was the one who submitted issue #34 but we also realized the plugin was now breaking editability for all form entries since the latest update to Gravity Forms.
You've made several people happy and grateful!

zackkatz committed:
Fix GF 2.5 styling issue

zackkatz committed:
Formatting only. Looks bad on multiple lines

zackkatz committed:
Update readme

Also, changed affiliate shortlink to full affiliate link to comply with WP.org requirements

zackkatz committed:
High-res GV logo

zackkatz committed:
Sanitize settings before storing in DB while allowing HTML

zackkatz committed:
Fix link to add-on in GF 2.5

zackkatz committed:
Esc more admin_urls()

andjules commented:
Awesome, thank you both
